variables.tf

```diff
@@ -2,19 +2,29 @@ variable "app_name" {
   type        = string
   description = "Scheduled Fargate Application name."
 }
-variable "ecs_cluster_name" {
-  type        = string
-  description = "ECS Cluster name to host the scheduled fargate task. Defaults to creating its own cluster."
+variable "existing_ecs_cluster" {
+  type = object({
+    arn = string
+  })
+  description = "Existing ECS Cluster configuration to host the fargate scheduled task. Defaults to creating its own cluster."
   default     = null
 }
-variable "schedule_expression" {
-  type        = string
-  description = "When to execute this fargate task. Use 'cron()' or 'rate()' At least one of schedule_expression or event_pattern is required."
+variable "schedule" {
+  type = object({
+    expression = string
+    timezone   = optional(string, "America/Denver")
```

Comment:
I'm wondering about this default. Do we want to assume it? And what does it mean if we use `rate()` but have a timezone?

Reply:
I believe it probably ignores the time zone if you use `rate()`. I'm fine if we don't want to assume this timezone, I figured that in most cases we'll want something run at 1:30am (or whatever time) in our timezone

Reply:
I personally would rather just not have a default.

Reply:
Fixed. No default